common/scripts/beachball-changelog-only.js

```diff
@@ -0,0 +1,26 @@
+// This is a workaround for generating changelog without bumping
```

Comment:
would be good to have a couple smoke unit-tests for this script

Reply:
(but realize it's a non-trivial amount of work given the directory this lives in)